openml/extensions/sklearn/extension.py

```diff
@@ -307,11 +307,16 @@ def _deserialize_sklearn(
                     rval = self._deserialize_rv_frozen(value)
                 elif serialized_type == "function":
                     rval = self._deserialize_function(value)
-                elif serialized_type == "component_reference":
+                elif serialized_type in ("composition_step_constant", "component_reference"):
```

Comment:
Are these scikit-learn names?

Reply:
No, these are magic constants used by us -> I should actually have them as magic constants at the top of the file.